privacyidea/lib/tokenclass.py

```diff
@@ -190,9 +190,9 @@ def set_user(self, user, report=None):
         :return: None
         """
         (uid, resolvertype, resolvername) = user.get_user_identifiers()
-        self.token.resolver = resolvername
-        self.token.resolver_type = resolvertype
-        self.token.user_id = uid
+        r = TokenOwner(token_id=self.token.id,
```

Comment:
I'm not so happy about this function because we change its meaning:
Before: ``set_user`` causes ``user`` to be the only owner of the token.
After: ``set_user`` causes ``user`` to be added as an owner of the token.

Even though we have the implicit precondition that we only call ``set_user`` if the token has no user, I would prefer to have one of the following safeguards here:
* We could rename the function to ``add_owner`` to make its meaning clear.
* We could assert that the token is unassigned and throw an error otherwise.
* We could unassign all current owners of the token.

What do you think?

Reply:
Again from an expectation level of the naming of the method I agree.

In the long run I think it makes sense to have a function ``add_user`` and ``set_user``.
Guess, what each function would be doing! ;-)
So I will rename this method to ``add_user``.

Will do 2019-02-14

Reply:
done in 49f1ac75